### PR TITLE
CU-g2t75w_Room-management-endpoints

### PR DESCRIPTION
**Pull request by ryandelvecchio**

### CODE REVIEW

**Comment by ryandelvecchio** [comment]
Task linked: [CU-g2t75w Room management endpoints](https://app.clickup.com/t/g2t75w)

**Review by carteralbrecht — CHANGES_REQUESTED** [review]
Is inclusion of client/debug.log intentional?

File: client/debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[1118/153743.090:ERROR:directory_reader_win.cc(43)] FindFirstFile: The system cannot find the path specified. (0x3)
```
Comment (carteralbrecht) [inline client/debug.log:1]: 🧐

Comment (ryandelvecchio) [reply client/debug.log:1]: Yeah I dont actually know where thats from lol

**Comment by carteralbrecht** [comment]
Lgtm but there are merge conflicts

**Review by carteralbrecht — APPROVED** [review]

**Pull request merged**